### DISCUSSION TITLE
Looking to Contribute

### DISCUSSION THREAD

**Discussion by btwomey873** [DaveSkender/Stock.Indicators#811] [Dev notes]
Hey Dave, 
I use your stock indicators quite a bit for my own applications and thought this would be a great place to start contributing.  Any ideas on where to start?

Thanks,
Ben

**Comment by DaveSkender** [comment]
Here are the steps to get started:

1. Review our [contributing guidelines](https://github.com/DaveSkender/Stock.Indicators/blob/main/docs/contributing.md#readme) to get acquainted with our working model, tools, practices, and standards.
2. Review [open issues](https://github.com/DaveSkender/Stock.Indicators/issues) with particular attention to items with the `good first issue` label -- these are usually easier and an excellent way to familiarize yourself with the code.  `help wanted` labels are also good to target, but they may be more complex or problematic.
3. Review the [project backlog](https://github.com/users/DaveSkender/projects/1) to see what else is planned.  Priority in the board view is right-to-left, top-to-bottom; so we're interested in getting the right and top items done first.
4. Once you find the most important item that you think is a good one to take on, let everyone know when you are starting to work on it to avoid multiple redundant efforts.  If it's an open issue, just mention your intention in the issue.  If it's only a Note in the project, create a new Issue and mention your intention.

Other things to consider:
- If you have new ideas for work, post a new [Idea discussion](https://github.com/DaveSkender/Stock.Indicators/discussions/categories/ideas) so that we can review the idea before committing to work.
- If you want to pair program, to speed-up the learning curve, contact me on Twitter [@daveskender](https://twitter.com/messages/compose?recipient_id=27475431)

**Reply by DaveSkender** [reply]
Also, there’s general things to do that aren’t really in the backlog.  For example, the documentation site could use better side navigation for indicators and utilities, or the demo site needs more indicators and other updates.

**Reply by DaveSkender** [reply]
If you have a general area of interest, let me know and I can help narrow down a few things to choose.
